homeassistant/components/ruuvi_gateway/bluetooth.py

```diff
@@ -46,10 +46,10 @@ def __init__(
 
     @callback
     def _async_handle_new_data(self) -> None:
-        now = datetime.datetime.now()
+        now = datetime.datetime.utcnow()
```

Comment:
It would be better to use the dt helper here.

Reply:
dt_util.utcnow is your git grep target

Reply:
That will cause

> TypeError: can't subtract offset-naive and offset-aware datetimes

Reply:
Ideally the lib returns datetime objects that are offset aware but thats probably not a great change to make for a patch release as time always bites us somehow.

Reply:
I got rid of `datetime.timedelta` altogether since we do get raw UNIX timestamps from the library in any case.